app/res/xml/app_restrictions.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<restrictions xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
They're set via the EMM console that I've been working on - so by Google itself, essentially. Can read about them [here](https://developer.android.com/work/managed-configurations.html)